CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+###Added
```

Comment:
the changelog is sensitive and your build fails if it isn't formatted to a certain syntax. please add a space between the # and A so that it is `### Added`